Composer/cypress/integration/CreateNewBot.spec.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-context('Creating a new bot', () => {
+context.skip('Creating a new bot', () => {
```

Comment:
we shouldn't skip this

Reply:
We shouldn't, but these E2E tests need to be re-written to cover the new creation dialog (which also pulls in adapters under the same feature flag).

Reply:
Can we turn off the new creation dialog when this is run?

Reply:
I'll write some new E2E tests to replace these - there's not a lot here in the first place.

Reply:
Actually, turning off the dialog is a better idea. I'll do that.